### DISCUSSION TITLE
How to display out of range content in the window

### DISCUSSION THREAD

**Discussion by coolH-alt** [fltk-rs/fltk-rs#1329] [Q&A]
```
    let app = app::App::default().with_scheme(app::AppScheme::Gtk);

    let mut win = Window::default().with_size(800, 800).center_screen();

    let mut win2 = Window::new(50, 50, 300, 300, None);
    {
        let mut win3 = Window::new(200, 200, 300, 300, None);
        win3.set_color(Color::Red);
        win3.end();
    }

    win2.set_color(Color::Blue);
    win2.end();

    win.make_resizable(true);
    win.size_range(800, 800, 0, 0);
    win.end();
    win.show();

    app.run().unwrap();
```
I want to see the complete win3 displayed, and win3 is a sub element of win2

**Comment by MoAlyousef** [comment]
I think windows clip their children, you might want to replace it with a Group widget.

Sent from my iPhone

On 9 Dec 2022, at 12:54, coolH-alt ***@***.***> wrote:

﻿

    let app = app::App::default().with_scheme(app::AppScheme::Gtk);

    let mut win = Window::default().with_size(800, 800).center_screen();

    let mut win2 = Window::new(50, 50, 300, 300, None);
    {
        let mut win3 = Window::new(200, 200, 300, 300, None);
        win3.set_color(Color::Red);
        win3.end();
    }

    win2.set_color(Color::Blue);
    win2.end();

    win.make_resizable(true);
    win.size_range(800, 800, 0, 0);
    win.end();
    win.show();

    app.run().unwrap();


I want to see the complete win3 displayed, and win3 is a sub element of win2
